src/blueapi/cli/rest.py

```diff
@@ -40,6 +55,19 @@ def get_device(self, name: str) -> DeviceModel:
     def get_state(self) -> WorkerState:
         return self._request_and_deserialize("/worker/state", WorkerState)
 
+    def set_state(
+        self,
+        state: Literal[WorkerState.RUNNING, WorkerState.PAUSED],
+        defer: Optional[bool] = False,
+    ):
+        return self._request_and_deserialize(
+            "/worker/state",
+            target_type=WorkerState,
+            method="PUT",
+            data={"new_state": state, "defer": defer},
+            raise_if=_allow_exceptions([400], _is_exception),
```

Comment:
In line 101 this function is the default anyways. I think it reads a bit neater to exclude it, i.e. 
`raise_if=_allow_exceptions([400])`

Reply:
Also; why are we allowing 400? Why aren't we raising on this exception, given in service/main.py it's the principal exception that can be raised from this endpoint?